### PR TITLE
Collect all the errors in a single file

### PR DESCRIPTION
**Pull request by raukadah**
It will allow developers to look for errors at one place and make it's easier to navigate over logs.

As a pull request owner and reviewers, I checked that:
- [ ] Appropriate testing is done and actually running

### CODE REVIEW

File: ci_framework/roles/artifacts/tasks/gather_errors.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+- name: Gather all errrors in a single file
+  ansible.builtin.shell: |
+    grep -rE '^[-0-9]+ [0-9:\.]+ [0-9 ]*ERROR ' {{ cifmw_artifacts_basedir }}/logs |
+    sed "s/\(.*\)\(20[0-9][0-9]-[0-9][0-9]-[0-9][0-9] [0-9][0-9]:[0-9][0-9]:[0-9][0-9]\.[0-9]\+\)\(.*\)/\2 ERROR \1\3/g" > /tmp/errors.txt;
+    mv /tmp/errors.txt {{ cifmw_artifacts_basedir }}/logs/errors.txt
```
Comment (cjeanner) [inline ci_framework/roles/artifacts/tasks/gather_errors.yml:6]: why not directly write in the final location?

**Comment by cjeanner** [comment]
Might good to get an example of the expected input and expected output - those regex are... complicated :).

**Comment by dasm** [comment]
@raukadah are you still working on this patch?

**Comment by cjeanner** [comment]
Closing PR and cleaning up associated branch. If this is still something we need, please feel free to re-open a new PR.

**Pull request closed without merging**